src/Microsoft.Health.SqlServer/Features/Schema/SchemaInitializer.cs

```diff
@@ -232,7 +253,7 @@ public void Start()
         {
             if (!string.IsNullOrWhiteSpace(_sqlServerDataStoreConfiguration.ConnectionString))
             {
-                Initialize();
+                InitializeAsync().Wait();
```

Comment:
can the start be async?

Reply:
We can have a separate user story to make this async. This Start is an implementation of the method in IStartable which is referenced in multiple files not related to sql server connection. Making this async would require changes in all those files as well. This would be absolutely a good idea to make them async at some point, sooner the better.

Reply:
The IStartable is right now now being called from IoC container which is sync call. There is a work item on this to make IStartable to be async but for now, I think this is fine.